packages/wallet/src/services/TransactionReemitter.ts

```diff
@@ -49,6 +49,8 @@ export const createTransactionReemitter = ({
 }: TransactionReemitterProps): Observable<Cardano.NewTxAlonzo> => {
   const volatileTransactions$ = merge(
     store.get().pipe(
+      // REVIEW: is it possible for the store to return undefined?
```

Comment:
No, it should never do that. When there's nothing stored it should complete without emitting anything